features/steps/command/shell.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Running shell commands
+
+  In order to test interactive command line applications
+  As a developer using Cucumber
+  I want to use the commands steps
+
+  Background:
+    Given I use a fixture named "cli-app"
+
+  Scenario: Running script
+    Given a file named "features/sehll.feature" with:
+
+    """
+   Feature: Running scripts
+    Scenario: Running ruby script
+      When I run the following script:
+        \"\"\"bash
```

Comment:
Please make sure, that indentation is similar to https://github.com/cucumber/aruba/blob/master/features/api/command/run.feature.